### ISSUE TITLE
Continue with the bug

### ISSUE THREAD

**Issue by nguy1072** [nguy1072/mtm6201-week2#2]
Dear Professor @kaura5, 

I have already followed your recommendation as justify-content: space-between for arranging the button, and it worked. However, the new problem comes out as the headers are not in the same line anymore (you can take a look at the image). I decided to use flex-grow: 1; for the emoji paragraph (emoji p) and it comes out perfectly. I don't know if I am allowed to use the flex-grow (emoji p) in this assignment.

Thank you so much, and hope you have a great day!

Best, 
Lyn 
![Problem](https://github.com/nguy1072/mtm6201-week2/assets/144856689/cc6e08ce-efa6-43b2-b102-939073131eb4)

**Comment by kaura5** [comment]
Yes, you can use any flex and grid property for this assignment.
